sucuri-cleanup.py

```diff
@@ -0,0 +1,22 @@
+# So far I'm mainly trying to collect the bits of code that I'm going to need for the core functionality of the script
+# this is by no means functional. And so far everything pretty much looks like a global. I'll work these bits down
+# into functions as I go, at least that's the plan so far ;)
+
+import sys
+import os
+import requests
+import re
+
+salts = requests.get('https://api.wordpress.org/secret-key/1.1/salt/')
+pwd = os.getcwd()
+install = pwd.split('/')[4]
+
+def salt_replace()
+    """
+    Replace the salts in the installs wp-config.php file & grab the prefix while we're there.
+    """
+    pass
+    wp_config = open("/nas/content/live/"+ install + "/wp-config.php", r+)
+    regex_list=["AUTH_KEY","SECURE_AUTH_KEY","LOGGED_IN_KEY","NONCE_KEY","AUTH_SALT","SECURE_AUTH_SALT","LOGGED_IN_SALT","NONCE_SALT"]
+    regex=re.compile()
+    for line in wp_config:
```

Comment:
So, I'd like to see the pass in the `for` loop so that the python parser doesn't throw up all over us until we write code for that... Here's what I had in mind:
```
def salt_replace()
    """
    Replace the salts in the installs wp-config.php file & grab the prefix while we're there.
    """
    # This is where the pass *used* to be...
    wp_config = open("/nas/content/live/"+ install + "/wp-config.php", r+)
    regex_list=["AUTH_KEY","SECURE_AUTH_KEY","LOGGED_IN_KEY","NONCE_KEY","AUTH_SALT","SECURE_AUTH_SALT","LOGGED_IN_SALT","NONCE_SALT"]
    regex=re.compile()
    for line in wp_config:
        pass # this is where I'd like to see it =]
```
You were close fersher, but lemme know what you think!

Cheers man!
Ryan